cmd/build-init/main.go

```diff
@@ -58,6 +62,18 @@ func main() {
 		logger.Fatal(err)
 	}
 
+	for _, c := range append(dockerCfgCredentials, dockerConfigCredentials...) {
```

Comment:
It would be interesting to pull this out to an an abstraction that can handle and load all three types of docker secrets in both rebase and build init.

Reply:
I agree. It would be nice to consolidate how we manage docker credentials